src/main/kotlin/no/nav/syfo/behandlerdialog/MeldingTilBehandlerService.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.syfo.behandlerdialog
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.behandlerdialog.api.MeldingTilBehandlerDTO
+import no.nav.syfo.behandlerdialog.database.createNewDialogmeldingOut
+import no.nav.syfo.behandlerdialog.database.getDialogmeldingOutForPersonident
+import no.nav.syfo.behandlerdialog.database.toDialogmeldingOut
+import no.nav.syfo.behandlerdialog.domain.DialogmeldingOut
+import no.nav.syfo.domain.PersonIdent
+
+class MeldingTilBehandlerService(
+    private val database: DatabaseInterface,
+) {
+    fun createMeldingTilBehandler(
+        personIdent: PersonIdent,
+        meldingTilBehandlerDTO: MeldingTilBehandlerDTO,
+    ) {
+        database.connection.use { connection ->
+            connection.createNewDialogmeldingOut(
```

Comment:
Bruke "Melding til Behandler" eller "Dialogmelding Out"? Synes det er litt vanskelig å bestemme seg for hva som er riktig, og om de kan "leve" sammen?

Reply:
Tror jeg foretrekker MeldtingTilBehandler som domeneklasse. "Out" blir litt mer "teknisk" - tenker det er viktigere å få fram at det er til behandler enn at den går "ut" 🤔

Reply:
MeldingOut er litt forvirrende fordi det ikke er entydig i hvilken retning, (en t-banestasjon har tog som kommer inn og skal ut i begge retninger, hvilken ut menes det da?). Må det være retning i navnet i det hele tatt, vi vet jo det er info som hører til utenfor appen siden det er DTO? Kaaaanskje vi bare trenger et domeneobjekt ala DialogmeldingDTO for begge retninger, og så kan det være en melding som har til-fra lapp og kanskje retning (sendt/mottatt) inni seg? 

Et annet problem er at ordet _til_ kan tolkes som en mapper, altså en melding som skal mappes om til en behandler. Og som Geir påpekte i sted, hvis man har en MeldingTilBehandler som faktisk skal mappes om til behandler, så blir det MeldingTilBehandlerToBehandler elns..

Hvis vi gjør et forsøk på å bruke kun ett domeneobjektet,  er det lettest hvis meldinger vi sender og meldinger vi mottar er noenlunde like. Det er de ikke helt, meldinger vi sender har en behandlerref (siden den skal bestilles sendt fra isdialogmelding), men det har ikke meldingene vi mottar fra padm2. Det er kanskje ikke nok til å ha eget domeneobjekter for inn og ut? Men hvis det er det, kanskje vi kan skille på de på at den ene er en meldingsbestilling, og det andre er en dialogmelding?